Avoid network test failure:
```
## --------------------------------- ##
## libdap 3.20.9 test suite: getdap. ##
## --------------------------------- ##
  1: getdap -d http://test.opendap.org/dap/data/nc/fnoc1.nc getdap-testsuite/fnoc1.nc.dds (pass) FAILED (getdapTest.at:34)

## ------------- ##
```
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -43,7 +43,7 @@ endif
 # TESTS = DASTest DDSTest EXPRTest DMRTest getdapTest
 
 check_SCRIPTS = $(DASTESTSUITE) $(DDSTESTSUITE) $(EXPRTESTSUITE) \
-	$(DMRTESTSUITE) $(GETDAPTESTSUITE)
+	$(DMRTESTSUITE)
 
 # Build the test drivers. The drivers all use the
 # subclassed types in Test*.cc and the TestTypesFactory.
@@ -117,12 +117,11 @@ DMRTESTSUITE = $(srcdir)/DMRTest
 GETDAPTESTSUITE = $(srcdir)/getdapTest
 
 check-local: atconfig atlocal $(DASTESTSUITE) $(DDSTESTSUITE) \
-$(EXPRTESTSUITE) $(DMRTESTSUITE) $(GETDAPTESTSUITE) 
+$(EXPRTESTSUITE) $(DMRTESTSUITE)
 	$(SHELL) '$(DASTESTSUITE)' $(TESTSUITEFLAGS)
 	$(SHELL) '$(DDSTESTSUITE)' $(TESTSUITEFLAGS)
 	$(SHELL) '$(EXPRTESTSUITE)' $(TESTSUITEFLAGS)
 	$(SHELL) '$(DMRTESTSUITE)' $(TESTSUITEFLAGS)
-	$(SHELL) '$(GETDAPTESTSUITE)' $(TESTSUITEFLAGS)
 
 # Make sure there are no spaces after the DASTESTSUITE names. jhrg 3/16/15
 # Delete the scripts here and not using CLEANFILES because this target is
